proposals/forms.py

```diff
@@ -630,3 +630,27 @@ def clean(self):
                     self.add_error('comments', _(
                         'Informatiebrief voor traject {} nog niet toegevoegd.').format(
                         study.order))
+    
+
+class TranslatedConsentForms(forms.ModelForm):
```

Comment:
So there's a bit of a weird interaction/bug with Django and how we build this multiple page form. If you're on a page like TranslatedConsentForms and there's an error in your input, you actually can't navigate away from it with the forward or next buttons. That's because those buttons try to save the input along the way, but that causes a Form error and Django says no. 

For this reason, SoftValidationMixin exists. We use it on proposal forms to solve this issue. It allows validation to continue and output its errors, but doesn't hard-prevent saving or form submission.

TranslatedConsentForms should use sfot validation as well, and it should mention both of its fields in `TranslatedConsentForms._soft_validation_fields`. But try to recreate the bug first so you get what I'm talking about